internal/docs/generated/livedocs/docs.go

```diff
@@ -91,7 +91,6 @@ functionality for polling a set of resources and computing the aggregate status
 for the set. For CRDs, there is a set of recommendations that if followed, will allow
 kpt live apply to correctly compute status.
 
-###
```

Comment:
Keep this, it is needed by the parser to not show these links in the built-in docs.

Reply:
I have updated the PR.